index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=poppins">
+    <link rel="stylesheet" href="./css/style.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <a class="logo" href="https://www.linkedin.com/in/jalal-ahmadi-58312b270/"><b>My Lgo</b></a>
+        <a href="#"><i class="material-icons">	list</i></a>
+    </header>
+    <section class="container">
+    
+        <div class="content">
+            <h1><b>I’m Alexandra.<br>
+
+                Glad to see<br>
+                you!</b></h1>
+            <p>I’m a software developer! I can help youbuild a product , feature or website Lookthrough some of my work and
+                experience! If you like what you see and
+                have a project you need coded, don’t
+                hesitate to contact me.<br>
+                <span>LETS CONNECT</span></p>
+        <div class="icons">
+        <ul>
+            <li><a class="icon1" href="https://twitter.com/JalalAh78951382"><img class="icon1" src="Vector (1).jpg" alt=""></a></li>
+            <li><a class="icon2" href="https://www.linkedin.com/in/jalal-ahmadi-58312b270/"><img class="icon2" src="Vector (2).jpg" alt=""></a></li>
+            <li><a class="icon3" href="https://www.linkedin.com/in/jalal-ahmadi-58312b270/"><img class="icon3" src="Vector (3).jpg" alt=""></a></li>
+            <li><a class="icon4" href="https://www.linkedin.com/in/jalal-ahmadi-58312b270/"><img class="icon4" src="Vector (4).jpg" alt=""></a></li>
+            <li><a class="icon5" href="#"><img class="icon5" src="Vector (5).jpg" alt=""></a></li>
+            <li><a class="icon6" href="https://github.com/jalal1379"><img class="icon6" src="Vector (6).jpg" alt=""></a></li>
+            <li><a class="icon7" href="#"><img class="icon7" src="Vector (7).jpg" alt=""></a></li>
+        </ul>
```

Comment:
- [ ] In implementing the previous reviewers requested changes in this [link](https://github.com/jalal1379/Microverse_portflaio/pull/1#discussion_r1215463443) you broke your design as shown in the image below. please before resubmitting for another review check to see that your design matches the provided figma template :+1:  :sparkles: 
- figma design
![cc](https://github.com/jalal1379/Microverse_portflaio/assets/89055502/bc8e9378-4355-4750-842e-fbc2d880beae)
- your design
![yc](https://github.com/jalal1379/Microverse_portflaio/assets/89055502/932c4266-4592-4503-8445-954b074eee4d)